gulpfile.js

```diff
@@ -219,6 +219,16 @@ const mappings = {
     'dir': 'powerbiembedded/lib',
     'source': 'arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json'
   },
+  'rdbms.mysql': {
+    'packageName': 'azure-arm-rdbms',
+    'dir': 'rdbmsManagement/lib/mysql',
+    'source': 'arm-rdbms/2017-04-30-preview/swagger/Rdms.MySQL.json'
+  },
```

Comment:
I'm not sure about this. It looks like `arm-rdbms` is checked into a private repo and is still not ready for the public repo `azure-rest-api-specs`.  I'm not sure if this repo hosts SDKs generated from private specs repository. Technically, I do not see a challenge as to why we cannot, but I'm not clear if that is the norm. @amarzavery ?